Scripts/LintLocalizableStrings.swift

```diff
@@ -105,14 +138,15 @@ let projectState: ProjectState = ProjectState(
     ),
     loadSourceFiles: targetActions.contains(.lintStrings)
 )
-print("------------ Processing \(projectState.localizationFiles.count) Localization File(s) ------------")
+print("------------ Processing \(projectState.localizationFile) ------------")
```

Comment:
Looks like this is crashing Xcode - it seems that `projectState.localizationFile` is actually convertible to a string and it results in a _giant_ output - changing this to the below fixes the issue:
```
print("------------ Processing \(projectState.localizationFile.path) ------------")
```

Reply:
This is really a nice catch! This slowed down the building process tremendously on my mac. After fixing it, the building time went back to normal! Thanks!!